### ISSUE TITLE
508-defect-2 [KEYBOARD]: Intercept Form - Focus must be managed when I close the modal or submit my input

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#26571]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Team PM will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Trevor

## User Story or Problem Statement

As a keyboard and screen reader user, I expect focus to be managed when I close the Intercept modal or submit my data.

## Details

Focus isn't managed when the user closes the Intercept modal (for both "Close" and "x"). After the user closes the modal, focus should be set to the `<body>` tag so pressing TAB once sets focus on the "Skip to content" link.

## Acceptance Criteria

- [ ] Upon closing the Intercept modal, focus is properly set to the body tag or the H1

**Comment by aubreyarcangel** [comment]
The problem statement was written for the old feedback button. With the new feedback at the bottom of the main content on the page, once a user closes the form, the focus is set back to the button. 
![focus](https://user-images.githubusercontent.com/11021491/158215909-62f40b42-2187-432d-9ac2-55dca18d08d7.gif)
